src/pretix/presale/ical.py

```diff
@@ -68,13 +68,16 @@ def get_public_ical(events):
         else:
             vevent.add('dtstart').value = ev.date_from.astimezone(tz).date()
 
-        if event.settings.show_date_to and ev.date_to:
-            if event.settings.show_times:
-                vevent.add('dtend').value = ev.date_to.astimezone(tz)
-            else:
-                # with full-day events date_to in pretix is included (e.g. last day)
-                # whereas dtend in vcalendar is non-inclusive => add one day for export
-                vevent.add('dtend').value = ev.date_to.astimezone(tz).date() + datetime.timedelta(days=1)
+        # ignore event.settings.show_date_to as ics needs an end for an event
```

Comment:
No, I don't think we should never ignore `show_date_to`. We currently never show the end date in this case, and I think it would be a big surprise for organizers if it suddenly does – in a pretty hidden place. :\

Reply:
Okay, I changed it, so when `show_date_to == False` then `ev.date_from` is used, even when an `ev.date_to` is available.